cmd/pokemon/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/ablarry/golang-bootcamp-2020/pkg/pokemons/api"
+	"github.com/gorilla/mux"
+)
```

Comment:
By convention, imported packages in a file should be separated into three separated blocks:

1st block: Golang library packages
2nd block: Your project's own dependencies (local packages)
3rd block: 3rd party packages

```suggestion
import (
	"log"
	"net/http"

	"github.com/ablarry/golang-bootcamp-2020/pkg/pokemons/api"
	
	"github.com/gorilla/mux"
)
```